entgql/internal/todo/ent/schema/todo.go

```diff
@@ -90,7 +90,7 @@ func (Todo) Edges() []ent.Edge {
 func (Todo) Annotations() []schema.Annotation {
 	return []schema.Annotation{
 		entgql.RelayConnection(),
-		entgql.QueryField(),
+		entgql.QueryField().Description("This the todo item"),
```

Comment:
I am not quite sure how to test the outcome of this, when running the tests locally, I have an error and do not know how to solve it.

![image](https://user-images.githubusercontent.com/18282328/184714256-49f5196e-cb52-4e2b-a578-f73b9c5fcdbd.png)